test/testbed/deploy_anvil.go

```diff
@@ -45,8 +47,15 @@ func NewAnvilContainerWithOptions(ctx context.Context, opts AnvilOptions) (*Anvi
 	// Generate a unique container name using timestamp to avoid conflicts in parallel tests
 	uniqueName := fmt.Sprintf("anvil-%d", time.Now().UnixNano())
 
+	// Build command with optional block time
+	// Note: foundry image uses ENTRYPOINT ["/bin/sh", "-c"], so we need a single shell command string
+	cmd := "anvil"
+	if opts.BlockTime > 0 {
+		cmd = fmt.Sprintf("anvil --block-time %d", opts.BlockTime)
```

Comment:
if we dont care about deterministic deployments/chain start, then we can just use `anvil --block-time %d --mixed-mining`

Reply:
https://github.com/Layr-Labs/eigenda/pull/2092/commits/cc866e2372f046dbcc006d1dbfc3d76b7e338d1f